scripts/kconfig/menuconfig.py

```diff
@@ -124,6 +124,25 @@ def _init_styles():
     global _INFO_BOT_SEP_STYLE
     global _INFO_HELP_STYLE
 
+    if sys.platform.startswith('win'):
```

Comment:
Something like `if platform.system() == "Windows"` might be better.

Reply:
This is taken from another toolkit, feel free to change